src/models/DTMFInput.ts

```diff
@@ -0,0 +1,16 @@
+export class DTMFInput {
+  inputIdentifier: string;
+  inputKey: string; // @TODO Change this to DTMFKey if required. Maybe unnecessary
```

Comment:
Yes, create an enum class for DTMFKey instead of a plain string, will provide more transparency

Reply:
Done